test/hasOwn_test.js

```diff
@@ -0,0 +1,60 @@
+goog.module('datalayerhelper.plain.testing.hasOwn');
+goog.setTestOnly();
+
+const {hasOwn} = goog.require('plain');
+
+describe('the isPlainObject method', function() {
+  /**
+   * Ensure that plain.hasOwn Determines if the value has a
```

Comment:
nit: don't capitalize "determines". I'd also rephrase this to something like: "Helper function used to ensure that plain.hasOwn...".